tmt/beakerlib.py

```diff
@@ -249,6 +255,9 @@ def fetch(self) -> None:
                 # Fallback to install during the prepare step if in rpm format
                 if self.format == 'rpm':
                     self.parent.debug(f"Repository '{self.url}' not found.")
+                    # Make note to do not attempt to clone it ever again
```

Comment:
shouldn't this be outside the if? the same might happen for fmf_id libraries and it will benefit from this speedup

Reply:
Good point. Let's make working for both cases. The problem  we raise different exception if it fmf so I need to find another way how to mark 'tried to clone but haven't succeed' than using format variable.

Reply:
Changed in 73b9191c, just a note that "real" tmt run will not benefit from caching invalid fmf-id urls as the very first error stops whole discover with the error.